Make it clear VUDRM tokens expire (#59)
Vualto/vudrm-public-docs: docs/source/VUDRM-token.md

- The encrypted DRM policy.
- A signed hash.

<div class="admonition danger">
    <p class="first admonition-title">Danger</p>

>     <p class="last">VUDRM tokens are designed to be single use and have a one to one relationship with the license request. Each license request must use a new vudrm token. VUDRM tokens will expire, the TTL is set at an account level and in production will be set to around 30 seconds. The time is counted from when the token was generated, the time displayed in the token. This is a separate time to the DRM policy times set in the encrypted DRM policy.</p>

Reviewer: ```suggestion
    <p class="last">VUDRM tokens are designed to be single use and have a one to one relationship with the license request. Each license request must use a new VUDRM token. VUDRM tokens will expire, the TTL is set at an account level and in production will be set to around 30 seconds. The time a token will expire is calculated by the time the token is generated plus the TTL; The time a token was generated is displayed in the token. This is a separate time to the DRM policy times set in the encrypted DRM policy.</p>
```
